CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+v0.7.0
+------
+* Update dependencies
+* Update function calls to support latest tilequeue changes
```

Comment:
* Update function calls to support latest tilequeue changes, including support for buffered MVT tile extents